content/docs/for-developers/sending-email/how-to-use-a-transactional-template-with-smtp.md

```diff
@@ -1,136 +1,130 @@
 ---
 seo:
-  title: How to use a Legacy Template with the SMTP API
-  description: Learn how to send a legacy template with the SMTP API.
-  keywords: SMTP, send email, integrate, building, filters, scheduling, substitution, suppression groups, unique arguments, recipients, legacy template
-title: How to use a legacy Template with the SMTP API
+  title: Sending a Transactional Template Email over SMTP
```

Comment:
We refer to these templates as legacy due to the introduction of Dynamic Templates.

Reply:
@Whatthefoxsays I see; should I update the Title to:

**Sending a Legacy Transactional Template Email over SMTP**